test/system/test_compilation/inline.h

```diff
@@ -0,0 +1,18 @@
+
+static inline void dummy_func_0(void) {
+	//NOP
```

Comment:
It's be good to make sure that it works when at least one of these inline functions actually DOES something... maybe even as simple as adding two numbers or calling another function?

Reply:
Sorry, a remnant of my first lazy implementation... fixed + added the different forms to the transform_inline_functions unit test as well because this is only checking the compilation and not what is actually output to the generated header file.